src/tests/multihost/sssd/testlib/common/qe_class.py

```diff
@@ -191,6 +191,18 @@ def dnf_uninstall(self, package):
                                raiseonerr=False)
         return cmd.returncode
 
+    @property    
+    def detect_files_provider(self):
+        """
+        Detect the SSSD's files provider feature 
+        :param: None
+        :return str: Return code of the ls command
```

Comment:
It returns 'bool', not 'str'.
It doesn't return 'code of the ls command', it returns 'if "files provider" is supported".

Reply:
updated.